course/views.py

```diff
@@ -24,6 +24,7 @@
 from .viewbase import CourseModuleBaseView, CourseInstanceMixin, EnrollableViewMixin
 
 
+
```

Comment:
Is this new line added with a reaon or by accident?

Reply:
Probably by accident, when I import the logging and was logging stuff there quickly to see that everything works